Extractor/Pushers/IPusher.cs

```diff
@@ -116,6 +127,17 @@ public interface IPusher : IDisposable
             return Task.FromResult((bool?)true);
         }
 
+        /// <summary>
+        /// Mark the given nodes as deleted in destinations.
+        /// </summary>
+        /// <param name="deletes"></param>
+        /// <param name="token"></param>
+        /// <returns></returns>
+        Task<bool> ExecuteDeletes(DeletedNodes deletes, CancellationToken token)
```

Comment:
Looks like neither of the 2 parameters here were used.

Reply:
This is the default implementation of an interface.